cpmonitor/models.py

```diff
@@ -174,56 +174,112 @@ class Meta:
     )
 
     cap_exists = models.BooleanField("Gibt es einen KAP?", default=False)
+    cap_rationale = models.TextField(
```

Comment:
Use `cap_exists_rationale`: I think it would make your life easier if you stick to the schema `<checklist_item>_rationale` and not shorten here. It makes the functions in `view.py` simpler and more robust, too.

Reply:
Good point, done